api/tests/python/end_points/payments/test_payments.py

```diff
@@ -88,123 +58,143 @@ def setup_draft_nr(client):
     return json.loads(post_response.data)
 
 
-def test_get_payment(client, jwt, app):
-    payment_id = 'abcd153'
-    request_uri = API_BASE_URI + payment_id
-    test_params = [{}]
-
-    query = build_test_query(test_params)
-    path = build_request_uri(request_uri, query)
-    log_request_path(path)
+@pytest.mark.skip
+def verify_fees_payload(payload):
+    assert isinstance(payload.get('filingFees'), float) is True
+    assert isinstance(payload.get('filingType'), str) is True
+    assert isinstance(payload.get('filingTypeCode'), str) is True
+    assert is_int_or_float(payload.get('futureEffectiveFees')) is True
+    assert is_int_or_float(payload.get('priorityFees')) is True
+    assert is_int_or_float(payload.get('processingFees')) is True
+    assert is_int_or_float(payload.get('serviceFees')) is True
+    assert is_int_or_float(payload.get('futureEffectiveFees')) is True
+    assert isinstance(payload.get('tax'), dict) is True
+    assert is_int_or_float(payload.get('total')) is True
 
-    response = client.get(path)
-    payload = json.loads(response.data)
 
+@pytest.mark.skip
+def verify_payment_payload(payload):
     assert isinstance(payload.get('id'), int) is True
-    assert isinstance(payload.get('invoices'), list) is True
+    assert isinstance(payload.get('nrId'), int) is True
+    assert isinstance(payload.get('payment'), dict) is True
+    assert isinstance(payload.get('sbcPayment'), dict) is True
+    assert isinstance(payload.get('statusCode'), str) is True
+    assert isinstance(payload.get('completionDate'), str) is True
+    assert isinstance(payload.get('token'), str) is True
 
 
-def test_create_payment(client, jwt, app):
-    draft_nr = setup_draft_nr(client)
+@pytest.mark.skip
+def execute_calculate_regular_fees(client):
+    """
+    1) Get the current the fees.
+    :param client:
+    :return:
+    """
+    headers = get_test_headers()
 
-    nr_num = draft_nr.get('nrNum')
-    request_uri = API_BASE_URI + nr_num
+    request_uri = API_BASE_URI + 'fees'
 
+    # Test regular submission
     path = request_uri
-    body = json.dumps(create_payment_request)
+    body = json.dumps({
+        'corp_type': 'NRO',
+        'filing_type_code': 'NM620',
+        'jurisdiction': 'BC',
+        'date': '',
+        'priority': ''
+    })
     log_request_path(path)
 
-    response = client.post(path, json=body)
+    response = client.post(path, data=body, headers=headers)
+
+    assert response.status_code == 200
```

Comment:
201?

Reply:
Returning 200 simply because there's actually nothing created despite it being a post, the fees are kinda the fees it's a calculation result as opposed to a resource. Just personal preference or whatever...